### PR TITLE
Fixing "mnemonic" and adding more words.

### PR DESCRIPTION
**Pull request by lh1008**
For discussion :) :+1:. 

These changes come from [Translation of user guides-PR #799](https://github.com/monero-project/monero-site/pull/799#discussion_r211088859)

### CODE REVIEW

File: spanish-terminology.md [hunk 1]
```diff
@@ -11,14 +11,14 @@
 | Keys | Claves | Se discutió "llaves", pero la comunidad Bitcoin ya utiliza la palabra "claves" |
 | Mine | Minar | |
 | Miner | Minero o Minador| |
-| Mnemonic seed | Semilla mnemotécnica | Semilla "mnemónica" es un argot, jerga. La palabra mnemotécnica viene de las siguientes palabras "Mnemo~ ⇒ Mnemósine, diosa griega de la memoria + técnica, que signifícan, técnicas para ayudar la memoria o, sencillamente, técnicas de memorización." Fuente [Mnemotecnia](https://www.mnemotecnia.es/inicio). |
+| Mnemonic seed | Semilla mnemómica | Mnemónica,co [RAE](http://dle.rae.es/?id=PRbUjYS) "Mnemo~ ⇒ Mnemósine, diosa griega de la memoria + técnica, que signifícan, técnicas para ayudar la memoria o, sencillamente, técnicas de memorización." Fuente [Mnemotecnia](https://www.mnemotecnia.es/inicio). |
```
Comment (ordtrogen) [inline spanish-terminology.md]: ojo al error de ortografía: pusiste "mnemómica"

La DRAE dice en cuanto a "mnemotécnica":
  "2. adj. Que sirve para auxiliar a la memoria."
lo que es la función de "mnemonic seed", 

en el caso de "mnemónica", no menciona ese significado.

File: spanish-terminology.md [hunk 2]
```diff
@@ -11,14 +11,14 @@
...
 | Open-source | Código abierto | |
 | Paper wallet | Monedero en papel | |
 | Pool | Pool | Se queda "pool" y no "piscina", ya que pool hace referencia a esfuerzo o valor compartido |
 | Receiver | Receptor | |
 | Ring signature | Firma de círculo | Se mantiene como firma de círculo por su uso en wikipedia |
 | Sender | Emisor | |
-| Solo mining | Minado en solitario | |
+| Solo mining | Minado en solitario / Minera en solitario | Estamos discutiendo la palabra "mining" ya que se utiliza y se encuentra que el uso de las palabras "minería en criptomonedas" tiene un uso generalizado en vez de "minado de criptomonedas". |
```
Comment (ordtrogen) [inline spanish-terminology.md]: Ojo! :
Minera --> Minería

**Comment by ordtrogen** [comment]
How about adding:

| Mining | Minería | |
| Pool mining | Minería en pool | |

while you're at it?

File: spanish-terminology.md (hunk 1 above)
Comment (lh1008) [reply spanish-terminology.md]: Yes it does, go to this link [RAE mnemónico/a](http://dle.rae.es/?id=PRbUjYS)

Comment (lh1008) [reply spanish-terminology.md]: btw, thank you for seeing the spelling error. Missed it while writing to fast, I'll be more careful in the future, thank you :).

File: spanish-terminology.md (hunk 2 above)
Comment (lh1008) [reply spanish-terminology.md]: Added! :) thank you.

**Pull request merged**